sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CResourceServerAutoConfiguration.java

```diff
@@ -82,7 +83,6 @@ public JwtDecoder jwtDecoder(JWTProcessor<SecurityContext> jwtProcessor) {
         if (!validAudiences.isEmpty()) {
             validators.add(new AADJwtAudienceValidator(validAudiences));
         }
-        validators.add(new AADJwtIssuerValidator());
```

Comment:
Why do we remove this?

Reply:
The *issuer* has been verified in **AADIssuerJWSKeySelector**, so we do not need to verify the *issuer* again in the follow-up.